server/api/scheduler.go

```diff
@@ -142,10 +142,23 @@ func (h *schedulerHandler) CreateScheduler(w http.ResponseWriter, r *http.Reques
 			return
 		}
 
-	case schedulers.GrantLeaderName:
-		h.addEvictOrGrant(w, input, schedulers.GrantLeaderName)
-	case schedulers.EvictLeaderName:
-		h.addEvictOrGrant(w, input, schedulers.EvictLeaderName)
+	case schedulers.GrantLeaderName, schedulers.EvictLeaderName:
+		storeID, ok := input["store_id"].(float64)
+		if !ok {
+			h.r.JSON(w, http.StatusBadRequest, "missing store id")
+			return
+		}
+		exist, err := h.AddEvictOrGrant(storeID, name)
+		if err != nil {
+			h.r.JSON(w, http.StatusInternalServerError, err.Error())
+			return
+		}
+		// we should ensure whether it is the first time to create evict-leader-scheduler
+		// or just update the evict-leader.
+		if exist {
+			h.r.JSON(w, http.StatusOK, "The scheduler has been applied to the store.")
```

Comment:
need to keep consistent with  https://github.com/tikv/pd/blob/cd0ffbafaa1adf9b3bc4d8cbe90e993c79f99e2b/pkg/schedule/schedulers/evict_leader.go#L395